dist/api_data.py

```diff
@@ -357,7 +352,17 @@ def __cmp__(self, other):
         the file version'''),
 ]
 
-lsm_meta = file_config + lsm_config + [
+# This is a set of configuration options that exist only on table and LSM
+# objects. They can be safely altered without exclusive access, as long as
+# the user doesn't need to see refreshed values without a restart.
+table_runtime_config = [
+    Config('app_metadata', '', r'''
+        application-owned metadata for this object'''),
+]
+
+# The table runtime metadata also applied to LSM trees, it currently contains
+# the user configurable app_metadata.
+lsm_meta = file_config + lsm_config + table_runtime_config + [
```

Comment:
Why do LSM trees need {{app_metadata}} if btrees don't?

Reply:
We thought table: and lsm: are the two main schemas users (for instance Mongodb) operate and set app_metadata on. So we decided to include LSM too.